README.md

```diff
@@ -25,11 +25,8 @@ Please read the [contribution guidelines](CONTRIBUTING.md).
 ### Testing
 * Testing Framework: [Mocha](http://mochajs.org/)
 * Assertion Library: [Chai](http://chaijs.com/)
-* Test Spies, Stubs and Mocks: [Sinnon](http://sinonjs.org/)
+* Test Spies, Stubs and Mocks: [Sinon](http://sinonjs.org/)
 * Test Coverage: [Istanbul](https://github.com/gotwarlost/istanbul)
-* ?? Test Runner: [Karma](https://karma-runner.github.io/1.0/index.html)
 
-     #### Testing Learning & Resources
-     * [Client-side testing with Mocha and Karma](https://sean.is/writing/client-side-testing-with-mocha-and-karma/)
-     * [Best Practices for Spies, Stubs and Mocks in Sinon.js](https://semaphoreci.com/community/tutorials/best-practices-for-spies-stubs-and-mocks-in-sinon-js)
-     * [Adding code coverage checking with Istanbul](https://egghead.io/lessons/javascript-how-to-write-a-javascript-library-adding-code-coverage-checking)
\ No newline at end of file
+### Resources
+See [RESOURCES.md](https://github.com/Moniarchy/floworky/blob/master/RESOURCES.md)
```

Comment:
`Moniarchy` needs to be changed to `GuildCrafts` - I *think* you can omit that though, and just use [RESOURCES.md]